packaging/packager_deb.sh

```diff
@@ -56,20 +74,28 @@ cp ../${NAME}_${VERSION}*.deb "$distro_root/guibot"
 apt-get -y install "$distro_root/guibot/"${NAME}_${VERSION}*.deb
 rm -fr "$distro_root/$NAME-$VERSION"
 
-# virtual display
+echo "------------- virtual display -------------"
 apt-get -y install xvfb vim-common
 export DISPLAY=:99.0
 Xvfb :99 -screen 0 1024x768x24 &> /tmp/xvfb.log  &
 touch /root/.Xauthority
 xauth add ${HOST}:99 . $(xxd -l 16 -p /dev/urandom)
 sleep 3  # give xvfb some time to start
 
-# unit tests
+echo " -------- set tesseract data environment -------- "
+export TESSDATA_PREFIX="/usr/share/tesseract-ocr/5/tessdata"
+echo "Tesseract data prefix: $TESSDATA_PREFIX"
+
+echo "------------ dump for environment variables and python path ------------"
+printenv
+python3 -c "import sys; print(sys.prefix)"
```

Comment:
Nice! Now that we have this dump of environment variables we don't need extra output for the DISABLE_AUTOPY one above as well as the extra `"Installing autopy..."` message which can also be traced back.